src/patient-registration/patient-registration-validation.tsx

```diff
@@ -14,4 +14,16 @@ export const validationSchema = Yup.object({
   monthsEstimated: Yup.number().min(0, 'Months cannot be less than 0'),
   telephoneNumber: Yup.string().matches(/^[0-9]*$/, 'Telephone number should only contain digits'),
   contactPersonPhone: Yup.string().matches(/^[0-9]*$/, 'Telephone number should only contain digits'),
+  contactPersonGivenName: Yup.string().when(['contactPersonPhone', 'contactPersonRelationship'], {
```

Comment:
This is a nice implementation for handling the validation` onSubmit`.
However, I've noticed 2 issues:
1. If the user inputs a middle name and nothing else, no error message is shown for given and family names being required. `contactPersonMiddleName` could be added to the `when` arguments.
2. No error message/validation is shown until the `Register Patient` button is clicked. We would want to inform the user as soon as they have 'touched' or inputted any data into` contactPersonPhone`, `contactPersonRelationship`, and `contactPersonMiddleName`. This could be achieved with `onBlur` or `touched` I think.